$DragonFly: src/gnu/usr.bin/gdb/gdb/patches/Attic/target.c.patch,v 1.3 2005/01/12 11:24:24 joerg Exp $

--- target.c	Mon Aug 26 12:18:33 2002
+++ target.c	Sun Oct 13 10:51:32 2002
@@ -1429,7 +1429,7 @@
   for (t = target_structs; t < target_structs + target_struct_size;
        ++t)
     {
-      if ((*t)->to_stratum == core_stratum)
+      if ((*t)->to_stratum == (kernel_debugging ? kgdb_stratum : core_stratum))
 	{
 	  runable = *t;
 	  ++count;
